.travis.yml

```diff
@@ -8,7 +8,7 @@ osx_image: xcode6.4
 
 env:
   matrix:
-    
+    - CONDA_R=3.3.1
```

Comment:
The CIs configuration should not be changed manually. Only `conda-smithy` should be used for that.

With that said I am not sure `conda-forge` support other version of R besides `3.3.2`. @johanneskoester do you have any plans on this? Is it worth at the moment?